documentation/airport-guides/egkk.md

```diff
@@ -0,0 +1,235 @@
+  
+  
+#### 1.1 Statistics
+* **Total passengers**: *46,576,473* (2019)
+* **Aircraft movements**: *284,987* (2019)
+* **Cargo**: *110,358 tonnes* (2019)
+  
+  
+### 2. Airspace
+In openScope, the airspace responsible for *Gatwick airport* is a combination of multiple sectors in reality. It encompasses large parts of the *London Terminal Control Area*, mainly to the south as the northern sectors are mainly used for traffic to and from *London Heathrow*, *London Luton* and *London Stansted*. The airspace extends from the surface up to maximum FL195, depending on the airspace section (see "Airspace Depiction"-view in openScope).  
+  
+#### 2.1 Sectors available in openScope
+The sim features a combination of these sectors:  
+
+```
+London Control
+Gatwick Director
+```
+  
+  
+### 3. Runways
+| Runway | Length (m) | (ft)   | Surface |
+|:------:|:----------:|:------:|:-------:|
+|08L/26R |2,565       |8,415   |Asphalt  |
+|08R/26L |3,316       |10,879  |Asphalt  |
+  
+*Gatwick airport* is using single-runway operations although it has two parallel runways. The close distance of the two runways (only 200m/656ft apart) is insufficient for simultaneous runway operations and therefore only the southern runway (08R/26L) is used for departures and landings during normal operations while the northern runway (08L/26R) is used as a taxiway and can only be used when the southern runway is out of use for any reason.  
+  
+**In openScope, Runway 26L is used for departures and landings, which is also the most common configuration at Gatwick (in use 74% of the time)**  
+  
+In the future *Gatwick airport* might "bring its existing standby runway into routine use". One scenario would see 08L/26R used for departing narrow-body aircraft only, while the longer 08R/26L would be used for wide-body take-offs and all landings. In longer term the airport remains interested in constructing a new runway to the south.  
+  
+  
+### 4. Instrument Procedures
+*Gatwick Airport* has a large number of SIDs and STARs, including both RNAV and non-RNAV procedures, as common for British airports. IFR flights will normally file for RNAV procedures although some procedures at *Gatwick* are only available as non-RNAV procedures.    
+  
+An overview of all available SIDs and STARs can be found below:  
+  
+#### 4.1 Standard Instrument Departures
+##### 4.1.1 Runway 08L
+| identifier | endpoint | RNAV | remark                                                |
+|:----------:|:--------:|:----:|-------------------------------------------------------|
+|   DVR2W    |    DVR   |   ❌  |                                                       |
+|   FRANE1W  |   FRANE  |   ❌  |                                                       |
+|   KENET3W  |   KENET  |   ❌  | restricted to traffic with destinations in UK or EIRE |
```

Comment:
Interesting choice to use Éire here. It's not wrong or anything, but it might better to use Ireland, given a lot of people are unlikely to recognise the Irish word for Ireland.

Reply:
I actually only copied this from the chart 😄 But I agree, Ireland is probably a better choice 👍

Reply:
Yeah I figured it had been taken from somewhere 😅